Code/Framework/AzCore/AzCore/Math/IntersectSegment.cpp

```diff
@@ -406,6 +406,13 @@ namespace AZ
         return ISECT_RAY_AABB_ISECT;
     }
 
+    Intersect::RayAABBIsectTypes Intersect::IntersectRayAABB2(const Vector3& rayStart, const Vector3& dirRCP, const Aabb& aabb)  
+    {
+        float start, end;
```

Comment:
nit: might be worth adding `_unused` to the end of these variables to indicate they're just throwaway values